dvc/utils/hydra.py

```diff
@@ -10,12 +11,38 @@
 from dvc.exceptions import InvalidArgumentError
 
 from .collections import merge_dicts, remove_missing_keys, to_omegaconf
-from .serialize import MODIFIERS
+from .serialize import DUMPERS, MODIFIERS
 
 if TYPE_CHECKING:
     from dvc.types import StrPath
 
 
+def compose_and_dump(
+    output_file: "StrPath",
+    config_dir: str,
+    config_name: str,
+    overrides: List[str],
+) -> None:
+    """Compose Hydra config and dumpt it to `output_file`.
+
+    Args:
+        output_file: File where the composed config will be dumped.
+        config_dir: Folder containing the Hydra config files.
+            Must be absolute file system path.
+        config_name: Name of the config file containing defaults,
+            without the .yaml extension.
+        overrides: List of `Hydra Override`_ patterns.
+
+    .. _Hydra Override:
+        https://hydra.cc/docs/next/advanced/override_grammar/basic/
```

Comment:
This link is dead?

Reply:
https://hydra.cc/docs/advanced/override_grammar/basic/